analyzer/library/egg/egg.go

```diff
@@ -0,0 +1,49 @@
+package egg
+
+import (
+	"os"
+	"strings"
+
+	"github.com/aquasecurity/go-dep-parser/pkg/egg"
+
+	"golang.org/x/xerrors"
+
+	"github.com/aquasecurity/fanal/analyzer"
+	"github.com/aquasecurity/fanal/analyzer/library"
+	"github.com/aquasecurity/fanal/types"
+)
+
+func init() {
+	analyzer.RegisterAnalyzer(&eggLibraryAnalyzer{})
+}
+
+const version = 1
+
+var requiredFiles = []string{".egg-info", ".egg-info/PKG-INFO"}
```

Comment:
Could you consider running it on Windows? We haven't done that, but we hope to do so in the future.